extensions/amazon-lambda-alexa/runtime/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-amazon-lambda-alexa-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-amazon-lambda-alexa</artifactId>
+    <name>Quarkus - Amazon Lambda Alexa - Runtime</name>
+    <description>Write AWS Lambda functions for Amazon Alexa</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-amazon-lambda</artifactId>
+        </dependency>
+    </dependencies>
```

Comment:
I'm surprised we don't have a dependency to some Alexa SDK?

Reply:
The Alexa SDK dependency is in the user project, since they need to subclass the SkillStreamHandler, and leverage other SDK dependencies to use Alexa with AWS Lambda.  Hence no need to include in the Quarkus dependencies.

Reply:
Added the dependency to the runtime BOM and also the runtime pom.  It needs to exclude logging, and an internal conflict.  User project doesn't require the explicit dependency now.